packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -36,7 +38,7 @@ export class WaferMap extends FoundationElement {
      * @internal
      * needs to be initialized before the properties trigger changes
      */
-    public readonly waferMapUpdateTracker = new WaferMapUpdateTracker(this);
+    public readonly waferMapUpdateTracker: WaferMapUpdateTracker = new WaferMapUpdateTracker(this);
```

Comment:
This change isn't needed right? You should only have to supply a type if it isn't set to a default.